configure.py

```diff
@@ -772,11 +772,11 @@
          'memory footprint, but also implies no just-in-time compilation ' +
          'support, thus much slower execution)')
 
-parser.add_argument('--v8-enable-object-print',
+parser.add_argument('--v8-disable-object-print',
     action='store_true',
-    dest='v8_enable_object_print',
-    default=True,
-    help='compile V8 with auxiliar functions for native debuggers')
+    dest='v8_disable_object_print',
+    default=False,
+    help='disable the V8 auxiliar functions for native debuggers')
```

Comment:
Since we're editing this line, might as well fix this typo:
```suggestion
    help='disable the V8 auxiliary functions for native debuggers')
```

Reply:
Thanks, I will fix typo.